test9.java

```diff
@@ -0,0 +1,12 @@
+class ThrowInFinally {
```

Comment:
RSPEC-8888: Test methods should be with format of :     testedMethod_testFlow_expectedResults
<!-- lookout comment separator -->
---
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._